configextractor.py

```diff
@@ -167,22 +169,24 @@ def classification_checker(res_section, parser_name, file_parsers):
 
 
 def subsection_builder(parent_section: ResultSection = None, fields: dict = {}):
+    # TODO: maybe iterate through fields and check if field in FIELD_TAG_MAP? not sure which is smaller
```

Comment:
fields would usually be smaller because it's the output from the parser

Reply:
Okay then I suggest something along the lines of:
```
for field_key, field_value in fields.items():
    if field_key in FIELD_TAG_MAP:
        tag = FIELD_TAG_MAP[field_key]
        ...
```

Reply:
done

Reply:
where?

Reply:
just pushed it now 
https://github.com/CybercentreCanada/assemblyline-service-configextractor/commit/18c0690310a356368cf839ae113c310971d4849d